model_path: "$(env HOME)/policy_1.pt" # model path
obs_dim: 47 # dimensions of observation space
action_dim: 12 # dimensions of action space
history_length: 50 # length of history frames
obs_history_length: 15 # length of history observations
lin_sensitivity: 0.2 # keyboard command sensivity for linear velocity
ang_sensitivity: 0.2 # keyboard command sensivity for angular velocity
